aibolit/metrics/coh/readme.md

```diff
@@ -0,0 +1,19 @@
+# Cohesion with LCOM4
+For `class A` with methods (functions: `func1`, `func2`, `func3`) and with attributes (`attr1`, `attr2`, `attr3`) we can build a graph `G1`:
+
+![asd](https://github.com/silverCase/aibolit/blob/metrics/aibolit/metrics/coh/G1.png)
+
+To measure the cohesion of class we will use `LCOM4` metric.\
+To understand `LCOM4` take a look at `LCOM3`:\
+`LCOM3` - calculates the number of connected component in the method graph. The methods in the graph are connected,
```

Comment:
> The methods in the graph

what does it mean?

Reply:
The vertices that represent methods in the graph